### PR TITLE
Feat: Add volatility and drift to the metrics module

### PR DESCRIPTION
**Pull request by Brendan-Lucas**
# Description
Add volatility and drift metrics and different time periods that the changes are expected in
left hand side marking hourly by default allows users to see different time periods over which the volatility and drift are expected to change over. 

## Sanity Check
 
Whole App:
Screen Shot 2022-11-03 at 7 59 46 PM

Metrics Module: 
Screen Shot 2022-11-03 at 7 59 38 PM

to SC check that when changing the time range from hourly to other values that the volatility and drift values change. If the time gets bigger, the vol and drift values should also be larger.

### CODE REVIEW

File: src/constants/constants.ts [hunk 1]
```diff
@@ -153,3 +155,10 @@ export const LOWER_TICK_COLOR = '#FF9433'
 export const UPPER_TICK_COLOR = '#F1DC00'
 export const LIQUIDATION_CONSTANT = 0.9375
 export const MAX_LEVERAGE = 10
+
+export const VolatilityResolutions = {
```
Comment (nickwang14) [inline src/constants/constants.ts:159]: Should this be an enum?

File: src/containers/MetricsModule/MetricsModule.module.scss [hunk 2]
```diff
@@ -66,6 +55,11 @@
       }
     }
 
+    .col {
```
Comment (nickwang14) [inline src/containers/MetricsModule/MetricsModule.module.scss:58]: Perhaps this should be in variables.scss

File: src/constants/constants.ts (hunk 1 above)
Comment (Brendan-Lucas) [reply src/constants/constants.ts:159]: I tried it out as an enum. it was bad. you cant use calculated numbers and I don't want to fight the typing this object makes sense and works.

File: src/containers/MetricsModule/MetricsModule.module.scss (hunk 2 above)
Comment (Brendan-Lucas) [reply src/containers/MetricsModule/MetricsModule.module.scss:58]: I agree * global, but I didn't want this edit to be responsible for changing that. I think I will move it though.

**Pull request merged**